src/main/java/com/ohmyclass/util/inteceptor/AccessInterceptor.java

```diff
@@ -0,0 +1,60 @@
+package com.ohmyclass.util.inteceptor;
```

Comment:
Why does this interceptor exist?
Delete it. (Only util.interceptor.AccessInterceptor; Keep the @Deprecated security.util.interceptor.AccessInterceptor).

Reply:
Change every single "FetchType.EAGER" to "FetchType.LAZY"
!